horace_core/sqw/@sqw/private/cut_sqw_main_single.m

```diff
@@ -167,10 +161,21 @@
 data_out.e(no_pix)=0;
 
 if opt.keep_pix
-    data_out.urange = urange_pix;
-    data_out.pix = PixelData(pix);
+    data_out.pix = PixelData(pix);    
+else
```

Comment:
what is this empty `else` for?

Reply:
probably left from the time when I was thinking about keeping pix_range only. Removed.